### PR TITLE
About: Updated licenses, authors for code and design and dates

### PR DESCRIPTION
**Pull request by carolanitz**
(closes #479)

### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: Resources/About Contents.html [hunk 1]
```diff
@@ -54,66 +59,69 @@ <h5>VLCFORIOSVERSION<br /><br />MOBILEVLCKITVERSION</h5>
...
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.videolan.org/developers/libdvbpsi.html">libdvbpsi</a><br />
 Copyright &copy; 2001-2017 VideoLAN and authors - <a href="#lgpl2">LGPLv2.1 or later</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://matroska.org">libebml</a><br />
 Copyright &copy; 2002-2017 Steve Lhomme - <a href="#lgpl2">LGPLv2.1 or later</a><br />
 <br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://libav.org">FFmpeg</a><br />
+Copyright &copy; 2000-2019 the FFmpeg developers - <a href="#lgpl2">LGPLv2.1 or later</a><br />
+<br />
```
Comment (fkuehne) [inline Resources/About Contents.html]: This should link to FFmpeg.

File: Resources/About Contents.html [hunk 2]
```diff
@@ -54,66 +59,69 @@ <h5>VLCFORIOSVERSION<br /><br />MOBILEVLCKITVERSION</h5>
...
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://aomedia.googlesource.com/aom">aom</a><br />
+Copyright &copy; 2016-2019, Alliance for Open Media - <a href="#bsd2clause">BSD-2</a><br />
+<br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://code.google.com/p/libass/">libass</a><br />
-Copyright &copy; 2006-2017 Grigori Goronzy and contributors - <a href="#isc">ISC License</a><br />
+Copyright &copy; 2006-2019 Grigori Goronzy and contributors - <a href="#isc">ISC License</a><br />
 <br />
-&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://libav.org">FFmpeg</a><br />
-Copyright &copy; 2000-2017 the FFmpeg developers - <a href="#lgpl2">LGPLv2.1 or later</a><br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://code.videolan.org/videolan/dav1d">dAV1d</a><br />
```
Comment (fkuehne) [inline Resources/About Contents.html]: Consider using "dav1d" as spelling like on the project page.

File: Resources/About Contents.html [hunk 3]
```diff
@@ -54,66 +59,69 @@ <h5>VLCFORIOSVERSION<br /><br />MOBILEVLCKITVERSION</h5>
...
 <br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/karlvr/XKKeychain">XKKeychain</a><br />
+Copyright &copy; 2014-2016 Karl von Randow - <a href="#mit">MIT License</a><br />
+<br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/xmartlabs/XLPagerTabStrip">XLPagerTabStrip</a><br />
+Copyright &copy; 2014-2019 xmartlabs - <a href="#mit">MIT License</a><br />
+<br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://aomedia.googlesource.com/aom">aom</a><br />
+Copyright &copy; 2016-2019, Alliance for Open Media - <a href="#bsd2clause">BSD-2</a><br />
+<br />
```
Comment (fkuehne) [inline Resources/About Contents.html]: Ouch. We should ship dav1d exclusively, but that's a separate issue.

File: Resources/About Contents.html [hunk 4]
```diff
@@ -153,9 +179,27 @@ <h5>VLCFORIOSVERSION<br /><br />MOBILEVLCKITVERSION</h5>
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.opus-codec.org">Opus</a><br />
 Copyright 2001-2013 Xiph.Org, Skype Limited, Octasic, Jean-Marc Valin, Timothy B. Terriberry, CSIRO, Gregory Maxwell, Mark Borgerding, Erik de Castro Lopo - <a href="#xiphbsd">Xiph.org BSD License</a><br />
 <br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://code.videolan.org/videolan/libplacebo">libplacebo</a><br />
+Copyright &copy; 2017-2019, Videolan, Niklas Haas and libplacebo Contributors - <a href="#lpgl2">LGPLv2.1 or later</a><br />
```
Comment (fkuehne) [inline Resources/About Contents.html]: VideoLAN

File: Resources/About Contents.html (hunk 1 above)
Comment (carolanitz) [reply Resources/About Contents.html]: will change, I just moved your code though ;)

File: Resources/About Contents.html (hunk 3 above)
Comment (carolanitz) [reply Resources/About Contents.html]: yeah we can do that in VLCKit and update the about screen, I'll file a ticket on VLCKit

**Comment by carolanitz** [comment]
merged with 7f592a954782464b1f2597df079b57c3f4c1e11d

**Pull request closed without merging**